spec/lib/common/json-schema-validator-spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+describe('JsonSchemaValidator', function () {
+    var validator;
+    var testSchema1 = { 
+        properties: {
+            repo: {
+                type: 'string',
+                format: 'uri'
+            }
+        }
+    };
+
+    helper.before();
+
+    before(function() {
+        var JsonSchemaValidator = helper.injector.get('JsonSchemaValidator');
+        validator = new JsonSchemaValidator();
+    });
+
+    it('should return true when validation pass', function () {
+        var result = validator.validate(testSchema1, { repo : '/172.31.128.1/mirrors' });
+        expect(result).to.be.true;
+    });
+
+    it('should throw validation error with incorrect data format', function () {
+        expect(function () {
+            validator.validate(testSchema1, { repo : 'abc' });
+        }).to.throw(/JSON schema validtion failed/);
```

Comment:
It's because I had typo in the code first , then copy & paste the string to unit test ...